src/doc/gui_manual/Operators/OperatorTypes/PICS/index.rst

```diff
@@ -0,0 +1,204 @@
+
+
+Source
+^^^^^^
+
+The set of points to seed the integral curves in the operator. See specific operator for details.
+
+Field
+"""""
+
+Sets the field type so that the native elements are used when interpolating the vector fields. Each operator provides the following options:
+
+Default
+    Use VisIt's native VTK mesh structure performing linear interpolation on the vector field.
+    
+Flash
+    Evaluates the velocity field via the Lorentz force. Parameters are:
+
+    * Constant - A constant multiple applied to the velocity.
+    * Velocity - When combined with Leap-Frog integration, this sets the initial velocity used in the integration.
+   
+M3D-C1 2D
+    Evaluates the 3D magnetic field via a 2D poloidal 6th order polynomial. Parameters are:
+
+    * Constant - A constant multiple applied to the perturbed part of the field.
+   
+M3D-C1 3D
+    Evaluates the 3D magnetic field via a 2D poloidal 6th order polynomial and 1D toroidal 4th order Bezier spline.
+    
+Nek5000
+    Evaluates the 3D vector field using `Nek5000 <https://nek5000.mcs.anl.gov/>`_ spectral elements.
+    
+Nektar++
+    Evaluates the 3D vector field using `Nektar++ <https://www.nektar.info/>`_ spectral elements.
+        
+NIMROD
+    Evaluates the 3D magnetic field via a 2D poloidal 3rd order Lagrange-type element and 1D toroidal finite Fourier series element (future implementation).
+
+
+Integration
+^^^^^^^^^^^
+
+Specify settings for the numerical integrator. See each operator for varied settings.
+
+
+Integrator
+""""""""""
+
+Sets the integration scheme. Options are:
+
+Forward Euler
+    The basic Euler integration with a fixed step length. This integrator is not recommended.
+
+Leap Frog
+    Simple method for numerical integration where both the position and velocity are updated using a fixed step length.
+
+Runge-Kutta-Dormand-Prince (RKDP)
+    A Runge-Kutta based scheme with adaptive step size control.
+    
+Adams-Bashforth (Multi-step)
+    A linear multistep integration using a fixed step length.
+    
+Runge-Kutta
+    Traditional 4th order Runge-Kutta with a fixed step length.
+    
+M3D-C1 2D Integrator (M3D-C1 2D fields only)
+    Uses Newton's method under the hood.
+    
+         
+Step Length
+"""""""""""
+
+The length of the integration step. Most integrators use a fixed step length. Runge-Kutta-Dormand-Prince uses adaptive step size, which can be clipped by this value.
+
+
+Tolerances
+""""""""""
+
+RKDP, Adams-Bashforth, and MD3-C1 make use of the tolerance options.
+    
+RKDP
+    Two tolerances are available - The adaptive step is set such that the maximum error at each step is less than the maximum between the absolute tolerance and the relative tolerance times the value of the vector field at the current point. The absolute tolerance can be truly absolute or relative to the bounding box.
+
+Adams-Bashforth
+    NOT REALLY CLEAR WHAT THIS IS BECAUSE AB USES A FIXED-STEP SIZE. MAYBE THIS IS A CLIP?
```

Comment:
I'd send an e-mail to Allen Sanderson and ask what this is.

Reply:
In this case yes a clip for the AB. Not sure what others do. Pinging @dpugmire as Dave put these together.

Reply:
@ARSanderson not sure if this was clear in my email, but this is what I was referring to regarding the tolerance. Does your comment about approaching a boundary still apply here?